lib/index.js

```diff
@@ -116,17 +117,18 @@ function addDataToSwaggerObject(swaggerObject, data) {
           }
           break;
         }
-        case 'tag': {
+        case 'tags': {
```

Comment:
Pushed an update. Used functions to avoid the problem with increasing the complexity.